force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -241,8 +309,12 @@ public with sharing class FlowControlLoopsCollections {
         Set<String> stringSet = new Set<String>();
 
         //add strings to the set
+        for(String str : inputList){
+            stringSet.add(str);
+        }
+        
 
-        return null; // Replace null with the variable you used to store the result
+        return stringSet; // Replace null with the variable you used to store the result
```

Comment:
Good work. A pro tip is that you can pass a list directly into the Set constructor. 

```
 public static Set<String> createStringSet(List<String> inputList) {
        return new Set<String>(inputList); // Replace null with the variable you used to store the result
    }
 ```